app/src/main/java/edu/uc/reedws/musiclink/service/PlaylistService.kt

```diff
@@ -18,12 +18,20 @@ class PlaylistService(application: Application) {
         getLocalPlaylistDAO()
     }
 
+    /**
+     * Fetch the playlists added by user
+     *
+     */
     fun fetchPlaylists(): LiveData<List<PlaylistDTO>> {
         return localPlaylistDAO.getPlaylists()
     }
 
+    /**
+     * Create the playlist and save the playlist into room database
+     *
+     */
     suspend fun createPlaylist(name: String): PlaylistDTO {
-        var playlist = PlaylistDTO(name)
+        val playlist = PlaylistDTO(name)
```

Comment:
I changed the variable type from var to val because the playlist variable will not be changed further in the code. I learned from a professor that it is always a good practice to switch from var to val if the data is not set to be changed in the future.